lib/metrics/drivers/statsd.rb

```diff
@@ -0,0 +1,46 @@
+module Metrics::Drivers
+  class Statsd
+    attr_reader :client, :template, :source_prefix
+
+    def initialize(client, template, source_prefix)
+      @client        = client        # The statsd client
+      @template      = template      # The template for the metric name
+      @source_prefix = source_prefix # Source prefix
+    end
+
+    def write(*instrumenters)
+      instrumenters.each do |instrumenter|
+        emit(instrumenter)
+      end
+      instrumenters
+    end
+
+    def emit(instrumenter)
```

Comment:
In fact, I kinda like the overall API in https://godoc.org/github.com/remind101/metrics better haha.